docs/core/extensions/options.md

```diff
@@ -27,6 +27,7 @@ Create the following `TransientFaultHandlingOptions` class:
 
 :::code language="csharp" source="snippets/configuration/console-json/TransientFaultHandlingOptions.cs" range="5-9":::
 
+<span id="options-class"></span>
```

Comment:
Was this line intentional?

Reply:
Yes, this allows for linking to this section without any visible indicators.